### PR TITLE
feat: add License File

### LINKED ISSUE #489: Add LICENSE file to our website repo
**Actual Behaviour**

 while reviewing pr: https://github.com/apache/apisix-website/pull/488, I found that we did not have LICENSE file in our website repo. And then I looked through the website of other Apache projects, they all have it in root dir.

see:
* https://github.com/apache/dolphinscheduler-website
* https://github.com/apache/skywalking-website
* https://github.com/apache/cassandra-website

**Expected Behaviour**

 Add a LICENSE file to our website repo root dir

### PR DESCRIPTION
**Pull request by KishaniKandasamy**
Fixes: #489 

Changes:

LICENCE file added

Hi everyone ,
Let me know your suggestions when you are available.

### CODE REVIEW

**Comment by KishaniKandasamy** [comment]
@liuxiran

**Comment by iamayushdas** [comment]
Reword this to make it semantic PR

**Review by liuxiran — APPROVED** [review]

**Comment by liuxiran** [comment]
@KishaniKandasamy thanks a lot, would you please change the commit message to `feat: xxx` so that we can pass the ci ?

**Review by bzp2010 — APPROVED** [review]

**Comment by KishaniKandasamy** [comment]
> @KishaniKandasamy thanks a lot, would you please change the commit message to `feat: xxx` so that we can pass the ci ?

Sure
